pkgs/applications/video/openshot-qt/default.nix

```diff
@@ -33,6 +35,8 @@ mkDerivationWith python3.pkgs.buildPythonApplication {
 
   buildInputs = [
     gtk3
+    qtwayland
+    wayland
```

Comment:
Please use `lib.optionals stdenv.isLinux`

Reply:
Is Wayland still bugged outside Linux?

Reply:
Not on darwin. We currently don't have a good way to represent this as qtwayland depends on `libdrm` (and we don't have recursive platform determining logic). Or `!stdenv.isDarwin`, whatever. This will very possibly be removed in future.